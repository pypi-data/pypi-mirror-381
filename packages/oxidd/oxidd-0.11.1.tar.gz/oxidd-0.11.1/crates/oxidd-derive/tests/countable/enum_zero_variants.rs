use oxidd_derive::Countable;

#[derive(Countable)]
enum Tag {}

fn main() {}
