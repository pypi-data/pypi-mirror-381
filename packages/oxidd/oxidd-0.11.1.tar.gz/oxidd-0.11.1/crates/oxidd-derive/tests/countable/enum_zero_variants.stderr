error: can only derive `Countable` for `enum`s with `repr(u8)`
 --> tests/countable/enum_zero_variants.rs:4:1
  |
4 | enum Tag {}
  | ^^^^^^^^^^^

error: cannot derive `Countable` for `enum`s with zero variants
 --> tests/countable/enum_zero_variants.rs:4:1
  |
4 | enum Tag {}
  | ^^^^^^^^^^^
